.rubocop.yml

```diff
@@ -0,0 +1,44 @@
+AllCops:
+    TargetRubyVersion: 2.4
+    Exclude:
+      - 'db/migrate/**/*'
+      - 'db/schema.rb'
+      - 'bin/**/*'
+
+Rails:
+    Enabled: true
+
+Bundler/OrderedGems:
+    Enabled: false
```

Comment:
Reason: This can be confusing when gemfiles get larger and it's difficult to see e.g. the external dependencies of the project